SocialCareCaseViewerApi/V1/Controllers/SocialCareCaseViewerApiController.cs

```diff
@@ -388,19 +388,18 @@ public IActionResult PostWarningNote([FromBody] PostWarningNoteRequest request)
         }
 
         /// <summary>
-        /// Get warning notes by person id
+        /// Get all warning notes created for a specific person
         /// </summary>
-        /// <param name="request"></param>
         /// <response code="200">Success. Returns warning notes related to the specified ID</response>
         /// <response code="404">No warning notes found for the specified ID</response>
-        [ProducesResponseType(typeof(List<WarningNote>), StatusCodes.Status200OK)]
+        [ProducesResponseType(typeof(ListWarningNotesResponse), StatusCodes.Status200OK)]
         [HttpGet]
-        [Route("warningnotes/{id}")]
-        public IActionResult GetWarningNote([FromQuery] GetWarningNoteRequest request)
+        [Route("residents/{personId}/warningNotes")]
+        public IActionResult ListWarningNotes(long personId)
```

Comment:
To me `GetWarningNotes` makes more sense, the function name describes what it is doing, List is referring to the type being returned and not the functionality?

You might disagree though which is ok.

Reply:
I can see both 🤔 
I was thinking from the perspective of `List`as a verb. So this lists all the warning notes created for a person, and maybe there would be a separate action to get a single warning note, but I don't have any strong opinions on this aside from whatever would clearly convey the idea behind the code.

Reply:
I think this is fine.